### DISCUSSION TITLE
[24 Gennaio 2023] - Modifica CPU

### DISCUSSION THREAD

**Discussion by rimaout** [CS-Swap/Architettura-degli-Elaboratori#28] [Modifica Architettura]
<img width="945" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/0ea4d04a-8d32-44b2-818e-aef811142008">
<img width="933" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/98b72b03-7994-4c88-b807-255d85632e81">
<img width="933" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/3424de6d-2e95-4552-8ea7-3d1df005a0fd">

<details>
<summary>Soluzione Prof</summary>
<img width="976" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/40787d92-6e3b-46c7-a8e1-748267146e4a">
<img width="780" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/e603e964-3384-45ff-a526-963c893a6e4a">

**Comment by notedo** [comment]
![ESERCIZI - Architettura-48](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/9744a068-c4a6-476e-ad49-65a78ffef1f1)
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/8c04e1b8-7d13-4bfb-99c3-bd20daa57cf7)
